rfc/0060-mod-menu.md

```diff
@@ -0,0 +1,73 @@
+
+
+## Explanation
+
+### Why are we doing this?
+Native Quilt Support, which lets us add additional features from things like quilt.mod.json or Quilt Config without issue.
+Standardization, allowing an official solution compared to Fabric's unofficial Mod Menu
+
+### Why not a fork of Terraformer's Mod Menu?
+The author of the Mod Menu himself, Prospector, said that ModMenu is "janky" - it is likely not worth the effort to clean up code AND integrate with other Quilt Standards. 
+
+### How will we deal with Fabric Mods that require configs?
+We will supply the Mod Menu API to allow for backwards compatibility with configs in Fabric mods.
+
+
+## Drawbacks
+
+### Potentially Unnecessary
+   It maybe be considered to be unnecessary to make this as the current Mod Menu solution on Fabric works fine. It also may complicate the user experience to have multiple mod menu mods. 
+   
+Fabric's Mod Menu is a separate mod, however, so it can simply be removed if a user finds it distracting.
+
+### Removes User Choice
+
+Some people may not want to be forced upon a Mod Menu in that case we are taking away their choice on whether or not to have a Mod Menu. 
+    
+This, however, is almost a non-issue; mod menus are often lightweight, and they are often the best way to abstract access to a mod's config. 
+
+### Burden of Maintenance
+
+Some people may view this as yet another Libarary / Mod to maintain every update and push new features to. 
+
+This is probably the most serious issue - an official Quilt Mod Menu will require official support - that means PR triage, updating between versions, and ensuring that as changes occur to things like the QMJ, the Mod Menu is appropriately updated. We propose the creation of a new team, the Mod Menu team, that would be responsible for its management.
+
+
+
+## Rationale and Alternatives
+
+
+- This allows for features from Loader and QSL to be used within the mod more effectively, and it also allows for easier communication within our teams in designing necessary features
+- As for alternatives, we could wait for a Quilt Mod Menu to show up, we could (against the author's recommendation) fork the TerraformersMC Mod Menu (and still have to create a new team for support). The proposed path is the best option.
+
+
+## Prior Art
+
+- TerraformerMCs Mod Menu
+    - Benefits
+        - Very well designed UI. Its visually pleasing and easy to use.
+        - API to allow for support of other config systems
+        - Supports showing parent/JiJed mods
+    - Drawbacks
+        - No native Quilt support. Thus it cant have support for features in Quilt such as support for Loader Plugin loaded mods, Quilt Config, Contributor Roles.
```

Comment:
> Thus it cant have support for features in Quilt such as support for Loader Plugin loaded mods, Quilt Config, Contributor Roles.

This isn't true, contributor roles are already supported and it's not particularly difficult to support more Quilt features without being a quilt-native mod.